app/src/main/java/org/mozilla/fenix/components/NotificationManager.kt

```diff
@@ -129,3 +133,10 @@ class NotificationManager(private val context: Context) {
         return this
     }
 }
+
+internal fun isValidTabSchema(tab: TabData): Boolean {
```

Comment:
Are we planning to use this just as a workaround until the main cause is fixed? (i.e. not syncing these schemas?)

Reply:
I think this would be a semi-permanent solution as it could be a very long-tail to ensure no version at some point could send a file:// across all browser platforms. Though I do think it's a relatively straightforward solution to just copy what desktop is doing for now but happy to brainstorm a different approach if we feel strongly against this.